sycl/include/CL/__spirv/spirv_types.hpp

```diff
@@ -40,12 +39,10 @@ enum MemorySemantics {
 // This class does not have definition, it is only predeclared here.
 // The pointers to this class objects can be passed to or returned from
 // SPIRV built-in functions.
-// Only in such cases the class is recognized as SPIRV type OpTypeEvent.
-class OpTypeEvent;
-
-// SPIRV type for sampler class
-class OpTypeSampler;
+// Only in such cases the class is recognized as SPIRV type __ocl_event_t.
```

Comment:
Silly point to bring up, but should this read "Only in such cases the class is recognized as OpenCL type __ocl_event_t" rather than SPIRV?

Reply:
I think we want SPIR-V translator to recognize `__ocl_event_t` as `OpTypeEvent`. Right?
Looks like another place where auto-replace should have been skipped.
@Fznamznon, please, rallback this change, if you and @agozillon do not object.

Reply:
Ah, I understand. I have no objections.

Reply:
I think we should fully rewrite this comment (lines 39-42) because there is no more OCL C++ classes without definition. We use another approach now: we re-use OpenCL special opaque types in SYCL device compiler but we added aliases for these types to avoid potential conflicts with C++ code. We need the following typedefs only for host compiler because it doesn't know anything about these types.